controllers/front/StoresController.php

```diff
@@ -151,8 +151,6 @@ protected function displayAjax()
 
         header('Content-type: text/xml');
         die($parnode->asXML());
```

Comment:
I think this can be useful to use ajaxDie method from Controller class instead of die, it can allow to use the hook actionBeforeAjaxDieStoresControllerdisplayAjax to modify xml

Reply:
good point, pr was updated.